include/glow/Quantization/Base/Base.h

```diff
@@ -99,6 +101,13 @@ QuantizationTransform32To8 quantizeScaleOffset32To8(float scale,
 TensorQuantizationParams chooseQuantizationParams(float min, float max,
                                                   Schema schema = Asymmetric);
 
+/// Row-wise quantize the tensor \p input. The param \p input is a 2-D
+/// tensor (i.e. M * N), \p scales and \p offsets are generated by each row of
+/// \p input, \p qData is 2D tensor quantized from \p input using \p scales and
```

Comment:
qData -> result?

Reply:
Here qData means quantized data. Since scales and offsets can be viewed as the results of this function, I think qData should be better.